ucs/clientlist.h

```diff
@@ -92,9 +92,9 @@ class Client {
 	void SendMailBoxes();
 	inline void QueuePacket(const EQApplicationPacket *p, bool ack_req=true) { ClientStream->QueuePacket(p, ack_req); }
 	std::string GetName() { if(Characters.size()) return Characters[0].Name; else return ""; }
-	void JoinChannels(std::string ChannelList);
-	void LeaveChannels(std::string ChannelList);
-	void LeaveAllChannels(bool SendUpdatedChannelList = true);
+	void JoinChannels(std::string channel_name_list, bool command_directed = false);
+	void LeaveChannels(std::string channel_name_list, bool command_directed = false);
+	void LeaveAllChannels(bool SendUpdatedChannelList = true, bool command_directed = false);
```

Comment:
Should these be `const std::string&` to avoid making a copy? Also, `send_updated_channel_list`?